storage/oasis/nodeapi/file/consensus.go

```diff
@@ -24,12 +24,15 @@ type FileConsensusApiLite struct {
 var _ nodeapi.ConsensusApiLite = (*FileConsensusApiLite)(nil)
 
 func NewFileConsensusApiLite(cacheDir string, consensusApi nodeapi.ConsensusApiLite) (*FileConsensusApiLite, error) {
-	db, err := pogreb.Open(cacheDir, &pogreb.Options{BackgroundSyncInterval: -1})
+	db, err := OpenKVStore(
+		log.NewDefaultLogger("cached-node-api").With("runtime", "consensus"),
```

Comment:
api uses the term "layer" for values that can be "consensus" or a runtime name

Reply:
I was on the fence what to use. We do use "layer" in the API, (I'm not a fan, even if I'm the one who introduced it :slightly_smiling_face:; I just don't have a better name), but not outside the API. Using "layer" here would make that more cohesive, but it would also mean that you cannot get "all" (most?) logs related to a runtime by filtering for `.runtime = "my_runtime_name"`, which is the log attribute that all/most other loggers use. I think the latter is more important.

Reply:
nvm I'll just mark this as resolved lol